### PR TITLE
Feature copy and paste

### PR DESCRIPTION
**Pull request by fedescarpa**
Fixes mumuki/mumuki-laboratory#1025

ezgif com-video-to-gif

### CODE REVIEW

File: src/javascripts/_pre-copy-paste.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  function copyToClipboard(text) {
+    var $temp = $("<textarea>");
+    $("body").append($temp);
+    $temp.val(text).select();
+    document.execCommand("copy");
+    $temp.remove();
+  }
+
+  function getEditor() {
+    return mumuki.page && mumuki.page.editors && mumuki.page.editors[0] && mumuki.page.editors[0]
```
Comment (luchotc) [inline src/javascripts/_pre-copy-paste.js]: Last condition is repeated twice

File: index.html [hunk 2]
```diff
@@ -147,5 +147,21 @@
...
   </div>
+
+  <br>
+
+  <div>
+    <textarea class="mu-paste-target" style="width: 100%; height: 250px"></textarea>
+
+    <br>
+
+    <pre><code>program {
```
Comment (luchotc) [inline index.html:158]: Missing indentation?

File: src/javascripts/_pre-copy-paste.js (hunk 1 above)
Comment (fedescarpa) [reply src/javascripts/_pre-copy-paste.js]: For more pleasure

File: index.html (hunk 2 above)
Comment (fedescarpa) [reply index.html:158]: no

**Pull request merged**